src/components/security_manager/src/crypto_manager_impl.cc

```diff
@@ -163,8 +163,13 @@ bool CryptoManagerImpl::Init() {
 #endif
   switch (get_settings().security_manager_protocol_name()) {
     case SSLv3:
+#if OPENSSL_VERSION_NUMBER < CONST_SSL_METHOD_MINIMAL_VERSION
       LOG4CXX_DEBUG(logger_, "SSLv3 is used");
       method = is_server ? SSLv3_server_method() : SSLv3_client_method();
+#else
+      LOG4CXX_DEBUG(logger_, "OpenSSL has no SSLv1.1.0, set SSLv1.0.2");
```

Comment:
@ValeriiMalkov Propose to change log message to the following one:
`LOG4CXX_DEBUG(logger_, "Current OpenSSL version is " << version << " and it has no SSLv1.1.0, set SSLv1.0.2 ");`

Reply:
@AByzhynar It was be delete, because SSLv3 is disable on ubuntu 16.04